commonmark/src/main/java/org/commonmark/node/Node.java

```diff
@@ -150,6 +150,21 @@ public void addSourceSpan(SourceSpan sourceSpan) {
         this.sourceSpans.add(sourceSpan);
     }
 
+    /**
+     * Check whether the given node is of the subset which count as "inline" nodes
+     * @param node Node to be checked for inline
+     * @return true if node is an inline node, otherwise false
+     */
+    public static boolean isInlineNode(Node node) {
```

Comment:
Could put this onto `Nodes` instead

Reply:
A search tells me I ended up not using this method, so I'll remove it. Thanks!